libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -466,6 +467,67 @@ export class SolanaWeb3Manager {
     })
   }
 
+  async purchaseContent({
+    id,
+    type,
+    splits
+  }: {
+    id: number
+    type: 'track'
+    splits: Array<{ address: string; amount: BN }>
```

Comment:
Do we want to support splits in V1 of this method? DN won't know how to support this, right?

Reply:
I think yes and no: I can add an error for if more than one is specified for now, and that way the interface won't change in the future but we're clear on not supporting multiple splits initially